frontend/rolecall/src/app/piece/piece_editor.component.ts

```diff
@@ -36,7 +45,8 @@ type WorkingPiece = Piece & {
 export class PieceEditor implements OnInit {
   dragAndDropData: DraggablePosition[] = [];
   currentSelectedPiece: WorkingPiece;
-  renderingPieces: WorkingPiece[];
+  workingPieces: WorkingPiece[];
+  renderingItems: RenderingItem[];
```

Comment:
Add comments for `workingPieces` and `renderingItems`, from variable names it's not obvious what they are for.

Reply:
Done.